app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -7,21 +7,19 @@ import android.view.Gravity
 import android.view.View
 import android.view.Window
 import android.widget.*
-import androidx.appcompat.app.AppCompatActivity
 import ch.sdp.vibester.BuzzerScoreUpdater
 import ch.sdp.vibester.R
 import ch.sdp.vibester.helper.GameManager
 import ch.sdp.vibester.helper.BuzzerGameManager
 import ch.sdp.vibester.model.Song
+import com.bumptech.glide.Glide
 import kotlin.collections.ArrayList
 import kotlin.collections.HashMap
 
-
-
 class BuzzerScreenActivity : GameActivity() {
 
-    private val MAX_N_PLAYERS = 4

```

Comment:
Do we have no more limitations on the maximum number of players?

Reply:
No, the limitation to 4 players is still there, but this val was only used in a check before I figured out how to directly fetch the player names from the intent, so it's not useful anymore.